src/docfx/build/page/BuildPage.cs

```diff
@@ -252,22 +252,21 @@ private static async Task<(List<Error> errors, JObject model)> LoadJson(Context
             var schemaValidationErrors = schemaTemplate.JsonSchemaValidator.Validate(obj);
             errors.AddRange(schemaValidationErrors);
 
-            // transform model via json schema
-            var (schemaTransformError, transformedToken) = schemaTemplate.JsonSchemaTransformer.TransformContent(file, context, obj);
-            errors.AddRange(schemaTransformError);
-            var pageModel = (JObject)transformedToken;
+            obj = (JObject)JsonUtility.ShallowClone(obj);
```

Comment:
nit: consider give it a different name if the variable is mutated:

```suggestion
            var xxx = (JObject)JsonUtility.ShallowClone(obj);
```

Reply:
using JsonUtility.Merge for shallow clone